rati_montreewat/week_02/tuesday/index.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html lang="en">
+<head>
+    	<meta charset="utf-8">
+    	<meta name="description" content="">
+    	<meta name="keywords" content="">
+
+		<title></title>
+
+		<link href="http://fonts.googleapis.com/css?family=Open+Sans:400,800" rel="stylesheet" type="text/css">
+		<link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <img src="images/larry_wall.png" alt="">
+  <div class="opacity">
+   <h1>Larry Wall</h1>
+   <p>Programmer 1954 - present</p>
+  </div>
+    <ul class="topnav">
+      <li><a href="biography.html">biography</a></li>
+      <li><a href="trivia.html">trivia</a></li>
+    </ul>
+  <div class="transboxright">
+    <p>&lt;quote&gt;<br>Real programmers can write assembly code in any language. <br>&gt;/quote&lt;
+    </p>
```

Comment:
Did you mean to use the HTML `<quote>` tags here (in which case you should not use the "&lt;" entities), or is it a deliberate effect?